tests/Microsoft.Identity.Test.Unit/Kerberos/KerberosSupplementalTicketManagerTests.cs

```diff
@@ -124,6 +124,7 @@ public void GetKrbCred()
             Assert.IsNotNull(krbCred);
         }
 
+#if !NETCOREAPP3_1

```

Comment:
This test is no longer functional on netstandard due to missing types needed for the test